internal/controller/controllers/crd_utils.go

```diff
@@ -0,0 +1,68 @@
+package controllers
+
+import (
+	"context"
+
+	adiiov1alpha1 "github.com/openshift/assisted-service/internal/controller/api/v1alpha1"
+	"github.com/sirupsen/logrus"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const APIVersion = "adi.io.my.domain/v1alpha1"
```

Comment:
I don't think you need this anymore, neither here nor in the tests where it's used.

Reply:
Done